ci/travis_script_rust.sh

```diff
@@ -35,6 +35,7 @@ cargo rustc -- -D warnings
 
 cargo build
 cargo test
+cargo bench
```

Comment:
It takes about a minute to run, so for now it's acceptable to run benchmarks on CI.

Reply:
Yes. We can cut it further down if it's needed. BTW: are we running benchmarks for other languages at the moment?

Reply:
AFAIK no